README.md

```diff
@@ -13,6 +13,16 @@ This demo shows how to use the Amazon Chime SDK to build an online classroom in
 - Log into your AWS account with an IAM role that has the **AdministratorAccess** policy.
 - Use the **us-east-1 (N. Virginia)** region of your AWS account.
 
+To deploy the classroom demo you will need:
```

Comment:
i think Cloud9 always has these tools installed. Move these prerequisites to the `### Option 2: Deploy from your local machine` section.

Reply:
Good callout! Moved!